AdblockPlusSafari/AdblockPlus.m

```diff
@@ -36,16 +36,25 @@ - (instancetype)init
 {
   if (self = [super init]) {
     _bundleName = [[[[[NSBundle mainBundle] bundleIdentifier] componentsSeparatedByString:@"."] subarrayWithRange:NSMakeRange(0, 2)] componentsJoinedByString:@"."];
-    NSString *group = [NSString stringWithFormat:@"group.%@.%@", _bundleName, @"AdblockPlusSafari"];
-    _adblockPlusDetails = [[NSUserDefaults alloc] initWithSuiteName:group];
+    _adblockPlusDetails = [[NSUserDefaults alloc] initWithSuiteName:self.group];
     [_adblockPlusDetails registerDefaults:
      @{ AdblockPlusActivated: @NO,
         AdblockPlusEnabled: @YES,
-        AdblockPlusAcceptableAdsEnabled: @YES}];
+        AdblockPlusAcceptableAdsEnabled: @YES,
+        AdblockPlusInstalledVersion: @0,
+        AdblockPlusDownloadedVersion: @1,
+        AdblockPlusFilterlists:
+          @{@"https://easylist-downloads.adblockplus.org/easylist_content_blocker.json":
+              @{@"filename": @"easylist"},
```

Comment:
The file names on the disk should also be changed accordingly, see https://issues.adblockplus.org/ticket/3093.